src/components/operators.jl

```diff
@@ -464,4 +464,18 @@ comp_prec(op::String) = get(AllPrecs, op, 0) === ComparisonOp || (length(op) > 1
 # Which ops have different precend when dotted?
 # [op for op in Symbol.(keys(CSTParser.AllPrecs)) if Base.isoperator(op) && Base.isoperator(Symbol(".", op)) && Base.operator_precedence(op) !== Base.operator_precedence(Symbol(".", op))]
 
+get_prec(op) = AllPrecs[maybe_strip_suffix(op)]
```

Comment:
Turns out something like `*ₛ` is a valid operator, so this adds support for that.